src/components/availability-label/availability-labels.tsx

```diff
@@ -56,6 +57,8 @@ export const AvailabiltityLabels: React.FC<AvailabilityLabelsProps> = ({
                   }
                 : undefined
             }
+            isbn={identifiers?.[0]?.value ?? ""}
+            isTrue={hasCorrectSource(["eReolen"], item)}
```

Comment:
Please rework the naming here. `isTrue` is not a very expressive name for what the prop represents.

If the prop is a way to force an availability label to have a specific value then one approach would be to change the value to `isAvailable`and make it an optional boolean. If it has a value then use that value for availability. If it is undefined then we look it up in the business systems.